CoursesManager.UI/Database/Procedures/Students/spStudents_Edit.sql

```diff
@@ -0,0 +1,30 @@
+DELIMITER $$
+CREATE DEFINER=`courses_manager`@`%` PROCEDURE `spStudents_Edit`(
+    IN p_id INT,
+    IN p_first_name VARCHAR(255),
+    IN p_last_name VARCHAR(255),
+    IN p_email VARCHAR(255),
+    IN p_phone VARCHAR(20),
+    IN p_address_id INT,
+    IN p_is_deleted BOOLEAN,
+    IN p_deleted_at DATETIME,
+    IN p_updated_at DATETIME,
+    IN p_insertion VARCHAR(255),
+    IN p_date_of_birth DATE
+)
+BEGIN
+    UPDATE students
+    SET
+        first_name = p_first_name,
+        last_name = p_last_name,
+        email = p_email,
+        phone = p_phone,
+        address_id = p_address_id,
+        is_deleted = p_is_deleted,
+        deleted_at = NOW(),
```

Comment:
deze is niet zo handig aangezien niet elke update een delete is. Misschien moet je daar nog even over nadenken. Volgens mij kun je wel if's maken in zo'n procedure

Reply:
IF statement toegevoegd als Is_deleted 1 is set deleted_at NOW() ELSEIF is_deleted 0 is set deleted_at NULL